lib/serverless-tools/deployer/ecr_pusher.rb

```diff
@@ -21,8 +22,12 @@ def push(local_image_name:)
       end
 
       def output
-        return {} unless image_tags.include?(tag)
-        asset
+        tags = image_tags
+        if tags.include?(tag)
+          return asset
+        end
+        Logging.logger.debug("Unable to find tag #{tag} in #{tags}")
```

Comment:
I'm going to set the environment variable to debug for the workflows on DLM repo.

Otherwise anyone using ST will get the log output and it's not really useful or user friendly in other circumstances.